addOns/automation/CHANGELOG.md

```diff
@@ -4,7 +4,12 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Fixed
+- When a Job is loaded via yaml and edited via Ui Dialog then the saved changes will be applied on the next job run
+- DelayJob will be verified on yaml load
 
+### Changed
+- When a Job is edited via Ui Dialog then the status will be set to NOT_STARTED
```

Comment:
`NOT_STARTED` → `Not started`, we should use plain English or what's actually shown to the user whenever possible in the release notes, they might not known details about the code (though here they should guess the constant is the label).